20-minimal/Dockerfile.c9s

```diff
@@ -0,0 +1,68 @@
+FROM quay.io/sclorg/s2i-core-c9s
```

Comment:
The same here.

Reply:
Same reason, it's closest to what we need and have available.